index.js

```diff
@@ -65,21 +65,10 @@ function createMainWindow() {
       nodeIntegration: false,
       plugins: true
     }
-  });
-
-  tuskWindow.loadURL(lastURL);
+  };
 
-  tuskWindow.on('close', e => {
-    if (!exiting) {
-      e.preventDefault();
-
-      if (process.platform === 'darwin') {
-        app.hide();
-      } else {
-        tuskWindow.hide();
-      }
-    }
-  });
+  const tuskWindow = new electron.BrowserWindow(Object.assign(defaultOptions, options));
```

Comment:
Here, we can avoid to directly override the `defaultOptions` object, and use it along with `options` to merge them against an empty object target: 

```js
const tuskWindow = new BrowserWindow(Object.assign({}, defaultOptions, options));
```

This way, after the merge, `defaultOptions` with still hold its original attributes.